.pre-commit-config.yaml

```diff
@@ -0,0 +1,54 @@
+---
+default_stages: [pre-commit]
+
+default_install_hook_types: [pre-commit, commit-msg]
+
+ci:
+  autoupdate_commit_msg: 'pre-commit: autoupdate hooks'
+  autofix_prs: false
+  # docker is not supported on pre-commit.ci
+  skip: [shellcheck]
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+        exclude: ^.*\.patch$
+      - id: mixed-line-ending
+        exclude: ^.*\.patch$
+
+  - repo: https://github.com/talos-systems/conform
+    rev: v0.1.0-alpha.30
+    hooks:
+      - id: conform
+        stages:
+          - commit-msg
+
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.3.0
+    hooks:
+      - id: codespell
+
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.35.1
+    hooks:
+      - id: yamllint
+        args: [-c=.yamllint]
+
+  - repo: https://github.com/3mdeb/hooks
+    rev: v0.1.2
+    hooks:
+      - id: namespell
+        args: [--fix]
+
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.41.0
```

Comment:
We have rev. `v0.36.0` in our `3mdeb/hooks` repo. Could you update it as well?

Reply:
`pre-commit-init.py` from 3mdeb/hooks automatically updates hooks to newest version during init.